model/team_test.go

```diff
@@ -112,24 +104,20 @@ var tReservedDomains = []struct {
 
 func TestReservedTeamName(t *testing.T) {
 	for _, v := range tReservedDomains {
-		if IsReservedTeamName(v.value) != v.expected {
-			t.Errorf("expect %v as %v", v.value, v.expected)
-		}
+		actual := IsReservedTeamName(v.value)
+		assert.Equal(t, actual, v.expected)
 	}
 }
 
 func TestCleanTeamName(t *testing.T) {
-	if CleanTeamName("Jimbo's Admin") != "jimbos-admin" {
-		t.Fatal("didn't clean name properly")
-	}
+	actual := CleanTeamName("Jimbo's Admin")
+	require.Equal(t, actual, "jimbos-admin", "didn't clean name properly")
 
-	if CleanTeamName("Admin Really cool") != "really-cool" {
-		t.Fatal("didn't clean name properly")
-	}
+	actual = CleanTeamName("Admin Really cool")
+	require.Equal(t, actual, "really-cool", "didn't clean name properly")
 
-	if CleanTeamName("super-duper-guys") != "super-duper-guys" {
-		t.Fatal("didn't clean name properly")
-	}
+	actual = CleanTeamName("super-duper-guys")
+	require.Equal(t, actual, "super-duper-guys", "didn't clean name properly")
```

Comment:
all of these should be updated to follow `require.Equal(t, expected, actual, ...)`

Reply:
fixed.